source/includes/_opportunities.md

```diff
@@ -0,0 +1,278 @@
+    "api_key": "YOUR API KEY HERE",
+    "owner_id":"5c10XXXXXXXXXXXXXXXXXXXX",
+    "name":"Opportunity Name",
+    "amount":"200",
+    "opportunity_stage_id":"5c14XXXXXXXXXXXXXXXXXXXX",
+    "closed_date":"2020-12-18",
+    "account_id":"5f06XXXXXXXXXXXXXXXXXXXX"
+}' "https://app.apollo.io/api/v1/opportunities"
+```
+
+> Sample response:
+
+```json
+{
+  "opportunity": {
+    "id": "5fdcXXXXXXXXXXXXXXXXXXXX",
+    "team_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+    "owner_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+    "salesforce_owner_id": null,
+    "amount": 200.0,
+    "closed_date": "2020-12-18T00:00:00.000+00:00",
+    "account_id": "5f06XXXXXXXXXXXXXXXXXXXX",
+    "description": null,
+    "is_closed": null,
+    "is_won": null,
+    "name": "Opportunity Name",
+    "stage_name": null,
+    "opportunity_stage_id": "5c14XXXXXXXXXXXXXXXXXXXX",
+    "salesforce_id": null,
+    "created_at": "2020-12-18T10:41:54.591Z",
+    "existence_level": "none",
+    "opportunity_contact_roles": []
+  }
+}
+```
+
+`POST https://api.apollo.io/v1/opportunities`
+
+| Parameter            | Description                                                                                                              |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| owner_id             | Owner ID . You can GET a list of possible users and their associated information from [Misc/Users](#get-a-list-of-users) |
+| name                 | Opportunity Name                                                                                                         |
+| amount               | The Amount of money involved in the Opportunity/ Deal                                                                    |
+| opportunity_stage_id | The ID of the current Stage (e.g. Prospecting, Closed Won, Closed Lost) of this Opportunity                              |
+| closed_date          | The date the Opportunity was closed                                                                                      |
+| account_id           | ID of the Account                                                                                                        |
+
+## Get all Opportunities
+
+> Sample request:
+
+```shell
+curl -X GET -H "Content-Type: application/json" -H "Cache-Control: no-cache" "https://api.apollo.io/v1/opportunities/search?api_key=YOUR_API_KEY_HERE"
+```
+
+> Sample response:
+
+```json
+{
+    "breadcrumbs": [],
+    "opportunities": [
+        {
+            "id": "5c14XXXXXXXXXXXXXXXXXXXX",
+            "team_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+            "owner_id": "5c100XXXXXXXXXXXXXXXXXXXX",
+            "salesforce_owner_id": "0051U000XXXXXXXXXX",
+            "amount": 90000.0,
+            "closed_date": "2018-08-27T15:00:00.000+00:00",
+            "account_id": "5c14223d945746a058358f93",
```

Comment:
The standard example would be better,  like `5c14XXXXXXXXXXXXXX`.

Reply:
oops I skipped that, I masked a whole lot of the id, my eyes skipped this